src/WebViewNativeComponent.android.ts

```diff
@@ -1,9 +1,3 @@
-import { requireNativeComponent } from 'react-native'
+const webView = require("./RNCWebViewNativeComponent").default
 
-const isFabricEnabled = global.nativeFabricUIManager != null;
-
-const webView = true ?
-    require("./RNCWebViewNativeComponent").default :
-    requireNativeComponent("RNCWebView")
-
-export default webView;
\ No newline at end of file
+export default webView;
```

Comment:
As explained [here](https://discord.com/channels/514829729862516747/702994331711045732/1002331710274404503), the `codegenNativeComponent` function calls the `requireNativeComponent` under the hoods.